src/commands/flags.ts

```diff
@@ -747,9 +747,9 @@ export class Flags {
     },
   };
 
-  static readonly hederaExplorerTlsLoadBalancerIp: CommandFlag = {
-    constName: 'hederaExplorerTlsLoadBalancerIp',
-    name: 'hedera-explorer-tls-load-balancer-ip',
+  static readonly hederaExplorerLoadBalancerIp: CommandFlag = {
```

Comment:
I like your name `mirrorStaticIP`.  Let us rename this to match:
```suggestion
  static readonly hederaExplorerStaticIp: CommandFlag = {
```

Reply:
Done